curp/src/server/cmd_worker/mod.rs

```diff
@@ -111,10 +111,10 @@ async fn worker_exe<
                 ce.execute(cmd, entry.index).await
             };
             let er_ok = er.is_ok();
-            cb.write().insert_er(entry.id(), er);
+            cb.write().insert_er(&entry.id(), er);
```

Comment:
We can pass the value to `insert_er` directly.

Reply:
done